### ISSUE TITLE
Build fails in RN0.76

### ISSUE THREAD

**Issue by johnf** [mateusz1913/react-native-avoid-softinput#211]
### Environment

Library version: 6.0.1
OS version: Android Pixel 7

### Affected platforms

- [X] Android
- [ ] iOS

### Current behavior

> Task :react-native-avoid-softinput:compileDebugKotlin FAILED
e: file:///home/johnf/work/gladly/mobile/node_modules/react-native-avoid-softinput/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt:35:16 Type mismatch: inferred type is ReactViewGroup? but ReactViewGroup was expected

### Expected behavior

No build error

### Reproduction

I think this isn't needed given it's a build error but I can create one if needed

**Comment by johnf** [comment]
FYI this patch gets it to compile but not sure if it breaks anything else (still working though other broken packages)
```diff
diff --git a/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt b/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt
index 0dae51e16646ee05035dd09331db2224b8cac2ea..5327b8f8276da49f2466e44b9c7f680cf9618c05 100644
--- a/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt
+++ b/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt
@@ -18,11 +18,6 @@ class AvoidSoftInputViewManager :
         return AvoidSoftInputView.NAME
     }
 
-    // ReactViewManager is not generic, so it doesn't let to pass any view that extends
-    // ReactViewGroup
-    // However, ReactViewManager does not use any delegate, so we can skip it and handle props here
-    override fun getDelegate(): ViewManagerDelegate<ReactViewGroup>? = null
-
     override fun createViewInstance(reactContext: ThemedReactContext): AvoidSoftInputView {
         return AvoidSoftInputView(reactContext)
     }
```

**Comment by johnf** [comment]
FYI: Not sure if correct but this allows it to compile

```diff
diff --git a/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt b/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt
index 0dae51e16646ee05035dd09331db2224b8cac2ea..35b4cb9482d177122c3616b070644476ead015af 100644
--- a/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt
+++ b/android/src/newarch/java/com/reactnativeavoidsoftinput/AvoidSoftInputViewManager.kt
@@ -30,7 +30,7 @@ class AvoidSoftInputViewManager :
     override fun prepareToRecycleView(
         reactContext: ThemedReactContext,
         view: ReactViewGroup
-    ): ReactViewGroup {
+    ): ReactViewGroup? {
         (view as AvoidSoftInputView).cleanup()
         return super.prepareToRecycleView(reactContext, view)
     }
```

**Comment by mateusz1913** [comment]
It's released in [v6.0.2](https://github.com/mateusz1913/react-native-avoid-softinput/releases/tag/v6.0.2) 🎉
